assets/blocks/reader-registration/index.php

```diff
@@ -90,6 +90,7 @@ function render_block( $attrs, $content ) {
 		'newsletterLabel'  => __( 'Subscribe to our newsletter', 'newspack' ),
 		'haveAccountLabel' => __( 'Already have an account?', 'newspack' ),
 		'signInLabel'      => __( 'Sign in', 'newspack' ),
+		'signedInLabel'    => __( 'An account was already registered with this email. Please check your inbox for an authentication link.', 'newspack' ),
```

Comment:
Not an addition of this PR, but default attributes are already set in `block.json`. Why should they be repeated here, @miguelpeixe ?

Reply:
These are placed as default values in case the attribute is set as an empty string, which does not fall back to the configured default value.